wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -177,7 +177,7 @@ October 25, 2016 (11:45 UTC)
 
 Game modes must be written as follows:
 
-- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!` (unofficial, but used to prevent ambiguity)
```

Comment:
wrong

Reply:
The file about ASC has been merged with #5246, the syntax should be good.